kong/cli/services/serf.lua

```diff
@@ -28,7 +28,11 @@ function Serf:_get_cmd()
   local cmd, err = Serf.super._get_cmd(self, {}, function(path)
     local res, code = IO.os_execute(path.." version")
     if code == 0 then
-      return res:match("^Serf v0.7.0")
+      local version_match = res:match("^Serf v0.7.0")
+      if not version_match then
+        logger:warn("Incompatible Serf version in the path. Required version is Serf v0.7.0")
+      end
+      return version_match
```

Comment:
I believe common practice would be to exit, indeed, and let the user re-order his or her `$PATH` if multiple Serf executables are indeed present. Otherwise, the warning, even if legit, is not that useful, and the user might not understand another version of Serf was used. Secondly, the exit error message (can't find Serf) is confusing too: it could find it, tho it just wasn't the right version.